### ISSUE TITLE
Possible to never cache pdf's?

### ISSUE THREAD

**Issue by dtu-compute** [dtu-compute/course-website#38]
How to reproduce

* visit https://01005-staging.compute.dtu.dk/enotes/introduction
* change some text in 
  * http://enote.compute.dtu.dk:14000
  * user:  test@dtu.dk
  * pass: test@dtu.dk
  * click Recompile
* Update Sharelatex on website

and the changes are not shown. If I however look at the website now from Incognito then I see the changes.

Would it be possible to never cache the pdf's?

**Comment by dtu-compute** [comment]
http://stackoverflow.com/a/4714980 mentions the following as a possible solution:

To prevent caching of dynamic content, all I use is this (and I haven't noticed any caching issues yet):
```
header('Cache-Control: no-cache, no-store, must-revalidate'); // HTTP 1.1
header('Pragma: no-cache'); // HTTP 1.0
header('Expires: 0'); // Proxies
```

**Comment by iainbryson** [comment]
For some reason GitHub doesn't seem to recognize the commit which should have fixed it.  I see it on both my local repo and on staging:

commit d3cc87e163ae4f0cd1e5f58a69febf4a5337dd20
Author: Iain Bryson <iain@iain-bryson.ca>
Date:   Mon Aug 15 18:43:25 2016 -0400

    Fix caching of eNotes for other courses. Fixes #38.


Do you know if it's fixed for you?

**Comment by dtu-compute** [comment]
Very strange!

The pdf's are not cached anymore, so I guess the ticket can be closed =)
